### DISCUSSION TITLE
Any way to determine url to raw logs of github actions?

### DISCUSSION THREAD

**Discussion by ivanixgames** [community/community#26474] [Actions]
<p>Hello,</p>
<p>I would like to provide link and possible line location of raw log file url that is found in the “View raw logs” pull down menu on the Actions  page.<br>
Is it possible to assible this link from GITHUB env variables during the action run?</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/ivanixgames">@ivanixgames</a> ,</p>
<p>It’s possible to get the existing raw log url. The URL follows the below format:</p>
<p><a href="https://github.com/%24%7B%7Bgithub.repository%7D%7D/commit/%24%7B%7Bcheckrun.headsha%7D%7D/checks/%24%7B%7Bcheckrun.id%7D%7D/logs" rel="nofollow noopener">https://github.com/<span class="math">{{github.repository}}/commit/</span>{{checkrun.headsha}}/checks/${{checkrun.id}}/logs</a></p>
<p>${{checkrun.headsha}}: can be get from checkrun rest <a href="https://developer.github.com/v3/checks/runs/#get-a-check-run" rel="nofollow noopener">api</a>:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/1/191f8dd70958c46423f1db44056a2834b7e8d43a.png" data-download-href="/uploads/short-url/3AfxxIaVqwlYiBb0dTNvQ43ZSxs.png?dl=1" title="checkrun_headsha.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/52772761/181097943-171219ec-8e9c-48dd-94ea-dfd4e7beb2d8.png" alt="checkrun_headsha.png" data-base62-sha1="3AfxxIaVqwlYiBb0dTNvQ43ZSxs" width="690" height="268" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/1/191f8dd70958c46423f1db44056a2834b7e8d43a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">checkrun_headsha.png</span><span class="informations">767×298 15.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Hope it helps.</p>

**Reply by miriam-lam-sndk** [reply]
Is there a way to get the logs using this url via the command line? 
I've tried some curl and gh api versions of this, but nothing seems to be working

**Comment by airtonix** [comment]
@miriam-lam-wdc an alternative might be this: https://github.com/marketplace/actions/job-summary-url
